android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -299,6 +299,21 @@ public void setHardwareAccelerationDisabled(WebView view, boolean disabled) {
     }
   }
 
+  @ReactProp(name = "androidLayerType")
+  public void setLayerType(WebView view, String layerTypeString) {
+    int layerType = View.LAYER_TYPE_NONE;
```

Comment:
layerType is none by default, according to [Android specs](https://developer.android.com/reference/android/view/View#attr_android:layerType)